pom.xml

```diff
@@ -98,6 +98,7 @@
 		<selenium_chrome_driver.version>3.14.0</selenium_chrome_driver.version>
         <selenium.version>4.17.0</selenium.version>
 		<sentry.version>7.1.0</sentry.version>
+        <apache_httpclient.version>5.2.1</apache_httpclient.version>
```

Comment:
why do we need this? I thought we already had another endpoint that is working.

Reply:
@pieterlukasse using apache httpclient instead of built-in jdk http client fixes e2e tests https://stackoverflow.com/questions/49119354/getting-java-net-httpretryexception-cannot-retry-due-to-server-authentication @haynescd contributed this fix

Reply:
Yea, its only added via test env